docs/workflow.md

```diff
@@ -31,3 +31,16 @@ To simulate Axe for Android (running on port 9051) for local testing of the app,
 ```sh
 yarn with:mock-axe-android start:electron:dev
 ```
+
+## Pull Request title tagging convention (Labelling commit messages)
+
+For every Pull Request, we prefer to add a tag in the PR title. This helps us in maintaining a good commit history as well as better documentation
+when creating Release notes.
+The convention that we follow is inspired from [SemVer](https://semver.org/) convention and is following:
```

Comment:
The linked page doesn't seem to mention `chore`, `feat`, or `refactor` - should we be linking somewhere else?

Reply:
The link is just to show that we are inspired from `semver` convention. I mentioned the workflow page, where we are trying to define what tags means for our codebase.